colorcoder/ColorUtility.java

```diff
@@ -0,0 +1,25 @@
+package colorcoder;
+
+import java.util.List;
+
+public class ColorUtility {
+
+    private static final int NUMBER_OF_MINOR_COLORS = MinorColor.getColors().size();
+
+    static ColorPair GetColorFromPairNumber(int pairNumber) {
+        int zeroBasedPairNumber = pairNumber - 1;
+        int numberOfMinorColors = MinorColor.MinorColors.values().length;
+        MajorColor majorColor = (MajorColor) getColorFromList(MajorColor.getColors(), zeroBasedPairNumber / numberOfMinorColors);
```

Comment:
is this form of down cast allowed as we are controlling who calls the private function